python/mxnet/ndarray.py

```diff
@@ -517,9 +517,9 @@ def reshape(self, shape):
 
     # pylint: disable= undefined-variable
     def broadcast_to(self, shape):
-        """Broadcasts an array to a new shape.
+        """Broadcasts the input array to a new shape.
```

Comment:
numpy has little page on the concept of broadcasting: https://docs.scipy.org/doc/numpy/user/basics.broadcasting.html

If what we do in mxnet is semantically equivalent to what numpy does, we could just point to that page.